MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: Selections in MAUI Calendar widget | Syncfusion
+description: Learn here all about Selections feature in Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Selections in MAUI Calendar (SfCalendar)
+The Selection mode is specified in the calendar property enumeration. Can select the dates by tap the cell in the calendar. `SfCalendar` provides three types mode such as `Single`, `Multiple` and `Range` Selection. The default `SelectionMode` is Single that allows the user to select one date at a time.
```

Comment:
The Selection mode is specified in the Calendar property enumeration. You can select the dates by tapping the cell in the calendar. `SfCalendar` provides three types of modes such as `Single,` `Multiple,` and `Range` Selection. The default `SelectionMode` is Single, which allows the user to select only one date at a time.

Reply:
Content modified